### DISCUSSION TITLE
Can I use the topic-operator to create/modify topics on a non-Strimzi-managed Kafka cluster?

### DISCUSSION THREAD

**Discussion by wushujames** [strimzi/strimzi-kafka-operator#6806] [Q&A] [answered]
Hi. I have an existing Kafka cluster that is running not-inside-Kubernetes.

I would like to make sure of the Strimzi CRDs to create topics in that existing external Kafka cluster.

Is it possible to use the Strimzi Topic Operator to do that? I would like it to only be used for creating/modify topics. I do not want to manage the Kafka cluster brokers using Strimzi. I also want the existing topics in the Kafka cluster to remain untouched by Strimzi.

Thanks.

**Comment by scholzj** [comment, marked as answer]
You can use it in what we call _standalone_ mode where you install it separately. It will require connection to Kafka and to ZooKeeper. There are some docs: https://strimzi.io/docs/operators/latest/full/deploying.html#deploying-the-topic-operator-standalone-str ... but there might be some limitations in the supported authentication mechanisms etc. which might not be needed when working with a Strimzi based Kafka cluster.

**Reply by wushujames** [reply]
This sounds like exactly what we want! Thanks, we'll take a look!
